### PR TITLE
Development Build

### PR DESCRIPTION
**Pull request by Logxn**
# ✔️ Ready To Merge

## 🔨 Changes
- [x] Fixed a typo that would show for people visiting via mobile devices
- [x] Updated the coloring of the infection numbers since the warning color was too bright
- [x] Improved the tooltip information for the 7-day-indication on desktop devices
- [x] Fixed typos in `ExpandablePageContent.tsx`
- [x] Updated the informations in the ExpandablePage to fit newer events 
- [x] Updated the packages and npm audit fixed a vuln (except typescript, since its buggy on the newest version - thanks)
- [x] Fixed `Formatter.tsx` to show the correct time of current data. (New data is supplied @ 3am instead of 12am)
- [x] `sevenDayIncident` is now a 2 fixed number again! >:(

## ✨ Addition
Mobile users won't be confused anymore when they see colored numbers.
`Tippy.js` is now being used to show some sweet tooltips.
And mobile users won't have to miss out on it!

### Regarding the extra information
I thought that it was at the time to update the text in the extra information section.
A lot of hypothetical theories that were mentioned there did actually happen. (I removed them)
Next I updated the information of americas case numbers since the grew so rapidly that they had to be updated.
I hope that the source I gave is "trustworthy" enough, if not I'll make sure to find another one.
Lastly I thought it was "worth mentioning" another german "star" that started to spread conspiracy theories

### Regarding the data date
RKI only provides new Data at 3:00am German time.
Thats why I thought it would be nice to show the correct time instead of 12 am.
Unfortunately I didn't think about the date, more percisely the day.
Example: If the user would visit the site at 12 am, he would see that the current data is from this day but from 3:00am.
Obviously we cannot look into the future, so I changed that.

### Regarding the toFixed(2) thing
The returned data (7-day-incident) is a number (and not a string ;) ) so I converted it.
There is no need to show a unnecessarely long number.

### Regarding the tooltips
Mobile users can now finaly see why we color different items.
I have added an extra package called `@tippyjs/react`.
I will still need to tinker around with the css part of that package, but for now it will do!

### CODE REVIEW

File: src/components/statistics/Statistics.tsx [hunk 1]
```diff
@@ -22,17 +24,30 @@ function Statistics() {
...
+  /*
+      The user visits on the 30.01 @ 1:00am
+      The website would show that the current data is from the 30.01 @ 3:00am.
+      This should be able to fix this problem
+  */
+  if(currentDate.getHours() < 3){
+      currentDate.setDate(currentDate.getDate() - 1);
+  }
+
   if(sevenDayIncident > 100){
```
Comment (janetschel) [inline src/components/statistics/Statistics.tsx:40]: Änder das so, dann sparst du dir ein else-statement:
```typescript
colorOfInfections = "#e0d916";
status = "Gefährlich";

if (sevenDayIncident > 100) {
    colorOfInfections =  "#c42929";
    status = "Kritisch";
} else if (sevenDayIncident < 50) {
    colorOfInfections =  "#43c429";
    status = "Ungefährlich";
}
```

Denk auch dran, dass man in Typescript Strichpunkte benutzt
Vielleicht switch-case? Weiß aber nicht, ob das kompakter ist

File: src/components/statistics/Statistics.tsx [hunk 2]
```diff
@@ -22,17 +24,30 @@ function Statistics() {
   Api.makeIncidentRequest(UrlConfig.urlIncident).then(result => setSevenDayIncident(result));
 
   const colorOfElement = infectionsDelta > 10000 ? "#c42929" : "black";
-  let information = "Derzeit werden die Infektionszahlen anhand des 7-Tage-Inzidenz-Wertes eingefärbt.\nAlles über 100 wird rot eingefärbt.\nAlles zwischen 50 und 100 wird gelb eingefärbt.\nAlles unter 50 wird grün eingefärbt.\n\nDerzeitiger Wert: "+sevenDayIncident;
-  
+  let status = "Kritisch";
   let colorOfInfections = "black";
```
Comment (janetschel) [inline src/components/statistics/Statistics.tsx:28]: Das brauchst du doch gar nicht (siehe if-statement unten).
`black` ist es nie, wird immer umgesetzt. also brauchst du das hier nicht, genauso wenig wie den status

File: src/components/statistics/Statistics.tsx [hunk 3]
```diff
@@ -1,4 +1,6 @@
 import React, { useState } from 'react';
+import Tippy from '@tippyjs/react';
+import 'tippy.js/dist/tippy.css';
```
Comment (janetschel) [inline src/components/statistics/Statistics.tsx:3]: Hab ich noch nie gesehen, muss man das CSS auch importieren

**Review by janetschel — CHANGES_REQUESTED** [review]
All in all guter PR, aber paar kleine Änderungen bzw. Clarifications für mich

File: public/index.html [hunk 4]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, minimun-scale=1, maximum-scale=1" />
     <link rel="icon" type="image/png" href="coronavirus.png"/>
+    <script src="https://unpkg.com/@tippyjs/react"></script>
```
[the remark this answers is not in the document]
Comment (Logxn) [reply public/index.html:7]: Hast recht, late night brain fart

File: src/components/statistics/Statistics.tsx (hunk 2 above)
Comment (Logxn) [reply src/components/statistics/Statistics.tsx:28]: Der Status wird im tooltip genutzt, check again

File: src/components/statistics/Statistics.tsx (hunk 3 above)
Comment (Logxn) [reply src/components/statistics/Statistics.tsx:3]: Das soll wohl optional sein.
Andernfalls kann mn auch `tippy.js/dist/headless` importieren um eignes css zu definieren.
Allerdings habe ich gerade noch einige probleme mit tippy's css

File: src/components/statistics/Statistics.tsx [hunk 5]
```diff
@@ -41,13 +56,24 @@ function Statistics() {
         <Typography className="HeaderSmall">Aktuelle Zahlen </Typography>
 
         <Typography className="AllInfectionsHeader">Alle Infektionen:</Typography>
+        <Tippy
```
[the remark this answers is not in the document]
Comment (Logxn) [reply src/components/statistics/Statistics.tsx:59]: Done

File: src/components/statistics/Statistics.tsx (hunk 1 above)
Comment (Logxn) [reply src/components/statistics/Statistics.tsx:40]: Done

**Comment by Logxn** [comment]
# 🔨 Revision 1
- [x] Removed unnecessary CDN url
- [x] Fixed tooltip font
- [x] Improved nesting

**Comment by Logxn** [comment]
# 🔨 Revision 1
- [x] Removed unnecessary CDN url
- [x] Fixed tooltip font
- [x] Improved nesting

**Comment by Logxn** [comment]
# 🔨 Revision 2
- [x] Fixed not showing the correct data of vaccinations

**Review by janetschel — APPROVED** [review]
LGTM

**Pull request merged**